File: app/src/main/java/com/fengmap/kotlindemo/MyApplication.java
package com.fengmap.kotlindemo;

import android.app.Application;

import com.mob.MobSDK;

import cn.jpush.android.api.JPushInterface;
import cn.smssdk.SMSSDK;

/**
 * Created by bai on 2018/12/6.
 */

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        MobSDK.init(this);
        JPushInterface.setDebugMode(true); 	// 设置开启日志,发布时请关闭日志
        JPushInterface.init(this);     		// 初始化 JPush
    }
}

File: app/src/main/java/com/fengmap/kotlindemo/MyApplication.kt
package com.fengmap.kotlindemo

import android.app.Application
import android.os.Environment
import com.fengmap.android.FMMapSDK

/**
 *
 * Created by bai on 2018/5/21.
 */
class MyApplication : Application() {
    override fun onCreate() {
        //初始化SDK
        FMMapSDK.init(this, Environment.getExternalStorageDirectory().getAbsolutePath()+"/kotlin/");
        super.onCreate()
    }
}